company-profiles/shattered-silicon.md

```diff
@@ -0,0 +1,35 @@
+# Shattered Silicon
+
+## Company blurb
+
+[Shattered Silicon](https://shatteredsilicon.net/) is a database consultancy company specialising in MySQL and MariaDB. We enable organisations to augment their database administration and maintenance capabilities. We can do this by leveraging the skills and experience of our highly qualified database experts.
+
+We are completely vendor-independent, which enables us to provide impartial [MariaDB and MySQL support](https://shatteredsilicon.net/mariadb-mysql-support/) and consultancy services. We provide our clients with the best, most appropriate solution possible for the problem they are solving. Our MySQL consulting experts are available 24/7 to help you increase performance, flexibility and business continuity of your database systems. We provide on-site [MariaDB and MySQL consulting](https://shatteredsilicon.net/postgresql-mariadb-mysql-consulting/) in Europe and remote DBA consulting world-wide.
+
+Our [MySQL experts](https://shatteredsilicon.net/mysql-experts/) recently released [Shattered Silicon Monitoring (SSM)](https://shatteredsilicon.net/mysql-monitoring-ssm/), a fork of Percona Monitoring and Management (PMM) v1. It contains additional features and fixes not present in PMM.
+
+## Company size
+
+5-10
+
+## Remote status
+
+Fully remote, flexible timezone, flexible work hours.
+
+## Region
+
+**Worldwide**
+
+## Company technologies
+
+MySQL, MariaDB, Linux, Prometheus, Grafana
+<https://github.com/shatteredsilicon>
+<https://shatteredsilicon.net/>
+
+## Office locations
+
+Our HQ is in Bulgaria, but all staff are internationally dispersed.
+
+## How to apply
+
+<contact@shatteredsilicon.net>
```

Comment:
Please add `Please email <contact@shatteredsilicon.net> to enquire about positions` or something similiar.This make it a tad clearer.